data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
+        description_completed= _ "Shamelessly Stole the Mother Gryphon's Egg"
+        icon="scenery/nest-full.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="scenery/nest-empty.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S10_Master_Pacifist"
+        name= _ "Master Pacifist"
+        name_completed= _ "Master Pacifist"
+        description= _ "Do not antagonize the Gryphons"
+        description_completed= _ "Did not antagonize the Gryphons"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/monsters/gryphon.png~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Strike_First_Strike_Hard"
+        name= _ "Strike First Strike Hard"
+        name_completed= _ "Struck First Struck Hard"
+        description= _ "Defeat the orcish commander on the northern bank of Abez"
+        description_completed= _ "Defeated the orcish commander on the northern bank of Abez"
+        icon="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/orcs/warlord-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Mirrored_Again"
+        name= _ "Mirrored Again"
+        name_completed= _ "Mirrored Again"
+        description= _ "Confront the scion of the Dark Queen once more"
+        description_completed= _ "Bested the scion of the Dark Queen yet again"
+        icon="units/unknown-unit.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="data/campaigns/Heir_To_The_Throne/images/units/human-princess.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+    [/achievement]
+
+    [achievement]
+        id="HttT_S11_Terror_to_the_Seas"
+        name= _ "Terror to the Seas"
+        name_completed= _ "Terror to the Seas"
+        description= _ "Defeat all spawned sea monsters in <i>Ford of Abez</i>"
+        description_completed= _ "Defeated all spawned sea monsters in <i>Ford of Abez</i>"
+        icon="units/monsters/kraken/kraken-defend1.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/monsters/kraken/kraken-defend2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S13_Northern_Rebirth"
+        name= _ "Northern Rebirth"
+        name_completed= _ "Northern Rebirth"
+        description= _ "Defeat all orc commanders in <i>Dwarven Doors</i>"
+        description_completed= _ "Defeated all orc commanders in <i>Dwarven Doors</i>"
```

Comment:
Are they in, or are they outside?

Reply:
> Are they in, or are they outside?

"in" if we're talking about the scenario, "outside" if we're talking about the geographical location? In which way is "*Dwarven Doors*" being used here?